packages/core/src/amazonq/webview/ui/tabs/constants.ts

```diff
@@ -6,6 +6,17 @@ import { TabType } from '../storages/tabsStorage'
 import { QuickActionCommandGroup } from '@aws/mynah-ui'
 import { userGuideURL } from '../texts/constants'
 
+const qChatIntroMessage = `Hi, I'm Amazon Q. I can answer your software development questions.
+  Ask me to explain, debug, or optimize your code.
+  You can enter \`/\` to see a list of quick actions. Use \`@\` to add saved prompts, files, folders, or your entire workspace as context.`
+
+export const qChatIntroMessageForSMUS = `Hi, I'm Amazon Q. I can answer your software development questions.\n\
+  Ask me to explain, debug, or optimize your code.\n\
+  You can enter \`/\` to see a list of quick actions. Use \`@\` to add saved prompts, files, folders, or your entire workspace as context.
+  You are now using Q free tier.\n\
+  `
+export const sageMakerUnifiedStudio = 'SageMakerUnifiedStudio'
```

Comment:
There is nothing Q-specific about this. Why is this in the Q constants file? It should live in `core/src/shared/constants.ts` at least, though there is probably a better place.